tsconfig.base.json

```diff
@@ -1,7 +1,6 @@
 {
   "compilerOptions": {
     "target": "es6",
-    "noImplicitAny": true,
```

Comment:
Why do we need to remove noImplicitAny, I like it because if I forget to add a type it screams at me.